Update doc abc

snozawa opened the pull request:
メモスライドを更新しつつ、doxygenのドキュメントからリンクをはられるようにしました。
よろしくお願いします。

k-okada commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/hrpsys-qnx/2182/
Test PASSed.

k-okada commented:
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.jsk.imi.i.u-tokyo.ac.jp:8080/job/hrpsys-qnx/2184/
Test PASSed.

snozawa commented:
こちらも何回かリスタートしたらtravisが通りました。

k-okada commented:
通りづらいのはどこ？
何回もリオープンするのではなくそちらを直すべきでは？

--
◉ Kei Okada

2015/07/19 19:39、Shunichi Nozawa <notifications@github.com> のメッセージ:

> こちらも何回かリスタートしたらtravisが通りました。
> 
> —
> Reply to this email directly or view it on GitHub.
>

snozawa commented:
`TEST_TYPE=work_with_315_1_10 TEST_PACKAGE=hironx-ros-bridge`
です。

snozawa commented:
ここ数日くらいの
https://travis-ci.org/fkanehiro/hrpsys-base/pull_requests
をみて、
https://travis-ci.org/fkanehiro/hrpsys-base/builds/71519308
https://travis-ci.org/fkanehiro/hrpsys-base/builds/71622456
などで
https://s3.amazonaws.com/archive.travis-ci.org/jobs/71622469/log.txt
のようにエラーがでています。

max time allottedでエラーなので時間が足りないのでしょうか。
```
<testsuite errors="1" failures="0" name="unittest.suite.TestSuite" tests="1" time="654.356">
  <testcase classname="rostest.runner.RosTest" name="testtest_hironx_ros_bridge_pose" time="654.3554">
    <error type="RLTestTimeoutException">test max time allotted
  File "/usr/lib/python2.7/unittest/case.py", line 327, in run
    testMethod()
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/runner.py", line 146, in fn
    self.test_parent.run_test(test)
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/rostest_parent.py", line 91, in run_test
    return self.runner.run_test(test)
  File "/opt/ros/hydro/lib/python2.7/dist-packages/roslaunch/launch.py", line 669, in run_test
    raise RLTestTimeoutException("test max time allotted")
    </error>
  </testcase>
  <system-out><![CDATA[[ROSTEST]setup[/opt/ros/hydro/share/hironx_ros_bridge/test/test-hironx-ros-bridge-pose.test] run_id[07fef250-2df7-11e5-9e4b-898626e31d9d] starting
[1mstarted roslaunch server http://testing-worker-linux-89e97461-1-14451-linux-15-71622469:37822/[0m
```

k-okada commented:
じゃあそれを増やしてみましょう

--
◉ Kei Okada


On Sun, Jul 19, 2015 at 7:53 PM, Shunichi Nozawa <notifications@github.com>
wrote:

> ここ数日くらいの
> https://travis-ci.org/fkanehiro/hrpsys-base/pull_requests
> をみて、
> https://travis-ci.org/fkanehiro/hrpsys-base/builds/71519308
> https://travis-ci.org/fkanehiro/hrpsys-base/builds/71622456
> などで
> https://s3.amazonaws.com/archive.travis-ci.org/jobs/71622469/log.txt
> のようにエラーがでています。
>
> max time allottedでエラーなので時間が足りないのでしょうか。
>
> <testsuite errors="1" failures="0" name="unittest.suite.TestSuite" tests="1" time="654.356">
>   <testcase classname="rostest.runner.RosTest" name="testtest_hironx_ros_bridge_pose" time="654.3554">
>     <error type="RLTestTimeoutException">test max time allotted
>   File "/usr/lib/python2.7/unittest/case.py", line 327, in run
>     testMethod()
>   File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/runner.py", line 146, in fn
>     self.test_parent.run_test(test)
>   File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/rostest_parent.py", line 91, in run_test
>     return self.runner.run_test(test)
>   File "/opt/ros/hydro/lib/python2.7/dist-packages/roslaunch/launch.py", line 669, in run_test
>     raise RLTestTimeoutException("test max time allotted")
>     </error>
>   </testcase>
>   <system-out><![CDATA[[ROSTEST]setup[/opt/ros/hydro/share/hironx_ros_bridge/test/test-hironx-ros-bridge-pose.test] run_id[07fef250-2df7-11e5-9e4b-898626e31d9d] starting
> �[1mstarted roslaunch server http://testing-worker-linux-89e97461-1-14451-linux-15-71622469:37822/�[0m
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fkanehiro/hrpsys-base/pull/723#issuecomment-122647233>
> .
>

snozawa commented:
https://github.com/start-jsk/rtmros_hironx/pull/374
で増やしてみました。
それいがいのエラーもあるようなので、もう少しみてみます。

k-okada commented:
これで通りやすくなるといいですね．以下のものはしょうがないですね．
```
[ 68%] Building CXX object CMakeFiles/HrpsysSeqStateROSBridge.dir/src/HrpsysSeqStateROSBridge.cpp.o
In file included from /home/travis/ros/ws_rtmros_hironx/src/rtm-ros-robotics/rtmros_common/hrpsys_ros_bridge/src/HrpsysSeqStateROSBridgeImpl.h:35:0,
                 from /home/travis/ros/ws_rtmros_hironx/src/rtm-ros-robotics/rtmros_common/hrpsys_ros_bridge/src/HrpsysSeqStateROSBridge.h:10,
                 from /home/travis/ros/ws_rtmros_hironx/src/rtm-ros-robotics/rtmros_common/hrpsys_ros_bridge/src/HrpsysSeqStateROSBridge.cpp:7:
/home/travis/ros/ws_rtmros_hironx/devel_isolated/hrpsys_ros_bridge/include/hrpsys_ros_bridge/idl/SequencePlayerServiceStub.h:12:50: warning: extra tokens at end of #ifndef directive [enabled by default]
/home/travis/ros/ws_rtmros_hironx/devel_isolated/hrpsys_ros_bridge/include/hrpsys_ros_bridge/idl/SequencePlayerServiceStub.h:13:50: warning: missing whitespace after the macro name [enabled by default]
g++: internal compiler error: Killed (program cc1plus)
Please submit a full bug report,
with preprocessed source if appropriate.
See <file:///usr/share/doc/gcc-4.6/README.Bugs> for instructions.
make[2]: *** [CMakeFiles/HrpsysSeqStateROSBridge.dir/src/HrpsysSeqStateROSBridge.cpp.o] Error 4
make[1]: *** [CMakeFiles/HrpsysSeqStateROSBridge.dir/all] Error 2
make: *** [all] Error 2
<== Failed to process package 'hrpsys_ros_bridge': 
  Command '/home/travis/ros/ws_rtmros_hironx/devel_isolated/rtmbuild/env.sh make -j1 -l1' returned non-zero exit status 2
```

snozawa commented:
他は、hrpsys-base/test/test-samplerobot-sequence.pyで頻繁にえらーがでます。
groupのところでposition limit overやerror limit overがでるときがときどきあります。
```
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
position limit over LARM_WRIST_R(25), qRef=7.63981e+162, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=7.7419e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.53681e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.75332e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=7.99098e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.49175e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.51612e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=8.31801e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.56927e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.52498e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.48727e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-9.57825e+166, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=-4.45704e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=-9.35134e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=-7.52494e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=-2.84881e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=-1.68593e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 0
position limit over LARM_WRIST_R(25), qRef=4.3361e+158, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=3.44739e+163, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=8.16032e+164, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.61631e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.47657e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.37548e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.43612e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.42672e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.39249e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.3215e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.20586e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.04131e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.82683e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.56418e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.10895e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.81905e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.62894e+164, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-5.20162e+164, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.05737e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.20213e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.3246e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.40924e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.45109e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.46023e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.24332e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.17318e+164, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.28355e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.00978e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.519e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.37729e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=3.38659e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.7567e+165, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.21576e+164, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.72253e+162, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.16005e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-8.35154e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.71712e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-1.73571e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=-3.79654e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.55344e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=3.67103e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=6.14775e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.10683e+168, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.26216e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.67209e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.1394e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=2.53031e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
Traceback (most recent call last):
  File "/home/travis/catkin_ws/install_isolated/share/hrpsys/test/test-samplerobot-sequence.py", line 12, in <module>
    samplerobot_sequence_player.demo()
  File "/home/travis/catkin_ws/install_isolated/share/hrpsys/samples/SampleRobot/samplerobot_sequence_player.py", line 303, in demo
    demoSetJointAnglesSequenceOfGroup()
  File "/home/travis/catkin_ws/install_isolated/share/hrpsys/samples/SampleRobot/samplerobot_sequence_player.py", line 287, in demoSetJointAnglesSequenceOfGroup
    checkJointAnglesBetween(p1, p0)
  File "/home/travis/catkin_ws/install_isolated/share/hrpsys/samples/SampleRobot/samplerobot_sequence_player.py", line 71, in checkJointAnglesBetween
    assert(ret is True)
AssertionError
position limit over LARM_WRIST_R(25), qRef=2.53031e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
HGcontroller0: onDeactivated(0)
terminate called after throwing an instance of 'CORBA::OBJECT_NOT_EXIST'
```

また、hironx_poseのところも下記のようにidlのところでエラーがでることがあります。
```
[rtmlaunch] Activate <- Inactive /localhost:2809/HrpsysJointTrajectoryBridge0.rtc
[rtmlaunch] Activate <- Inactive /localhost:2809/DataLoggerServiceROSBridge.rtc
[rtmlaunch] Activate <- Inactive /localhost:2809/SequencePlayerServiceROSBridge.rtc
[rtmlaunch] Activate <- Inactive /localhost:2809/ForwardKinematicsServiceROSBridge.rtc
[rtmlaunch] Activate <- Inactive /localhost:2809/StateHolderServiceROSBridge.rtc
[0m[ INFO] [1437416571.128315927, 83.599999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1762 Hz, dropped 201)[0m
[0m[ INFO] [1437416572.131268558, 84.609999999]: [HrpsysSeqStateROSBridge0] @onExecutece 0 is working at 201[Hz] (exec 1764 Hz, dropped 201)[0m
Traceback (most recent call last):
  File "/opt/ros/hydro/share/hironx_ros_bridge/test/test_hironx_ros_bridge_pose.py", line 118, in <module>
    rostest.rosrun(PKG, 'test_hronx_ros_bridge_pose', TestHiroROSBridgePose)
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/__init__.py", line 136, in rosrun
    suite = unittest.TestLoader().loadTestsFromTestCase(test)
  File "/usr/lib/python2.7/unittest/loader.py", line 56, in loadTestsFromTestCase
    loaded_suite = self.suiteClass(map(testCaseClass, testCaseNames))
  File "/opt/ros/hydro/share/hironx_ros_bridge/test/test_hironx_ros_bridge.py", line 49, in __init__
    self.robot.init()
  File "/opt/ros/hydro/lib/python2.7/dist-packages/hironx_ros_bridge/hironx_client.py", line 152, in init
    HrpsysConfigurator.init(self, robotname=robotname, url=url)
  File "/home/travis/catkin_ws/install_isolated/lib/python2.7/dist-packages/hrpsys/hrpsys_config.py", line 2059, in init
    self.setSelfGroups()
  File "/opt/ros/hydro/lib/python2.7/dist-packages/hironx_ros_bridge/hironx_client.py", line 373, in setSelfGroups
    self.seq_svc.addJointGroup(item[0], item[1])
  File "/home/travis/catkin_ws/install_isolated/lib/python2.7/dist-packages/hrpsys/SequencePlayerService_idl.py", line 179, in addJointGroup
    return _omnipy.invoke(self, "addJointGroup", _0_OpenHRP.SequencePlayerService._d_addJointGroup, args)
omniORB.CORBA.COMM_FAILURE: CORBA.COMM_FAILURE(omniORB.COMM_FAILURE_WaitingForReply, CORBA.COMPLETED_MAYBE)
cache found for file:///opt/ros/hydro/share/hironx_ros_bridge/models/kawada-hironx.dae
Warning: Model HiroNX has empty joint ID in the valid IDs.
Loading body customizer "/opt/ros/hydro/share/OpenHRP-3.1/customizer/libSampleCustomizer.so" for springJoint
cache found for file:///opt/ros/hydro/share/OpenHRP-3.1/idl/../../..//share/OpenHRP-3.1/sample/model/longfloor.wrl
[33m[rtmlaunch] Catch signal 'SIGINT', exitting...[0m
[33m[rtmlaunch] Catch signal 'SIGINT', exitting...[0m
[0m[ INFO] [1437416573.212322617, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (larm[0m
[0m[ INFO] [1437416573.214622279, 84.949999999]: [HrpsysSeqStateROSBridge] @onFinalize : HrpsysSeqStateROSBridge0[0m
terminate called after throwing an instance of 'CORBA::COMM_FAILURE'
terminate called after throwing an instance of 'CORBA::OBJECT_NOT_EXIST'
[0m[ INFO] [1437416573.226336870, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (rarm[0m
[0m[ INFO] [1437416573.238987756, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (head[0m
[0m[ INFO] [1437416573.252409911, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (torso[0m
[0m[ INFO] [1437416573.264743400, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (larm_torso[0m
[0m[ INFO] [1437416573.276628556, 84.949999999]: [HrpsysJointTrajectoryBridge0] @~jointTrajectoryActionObj (rarm_torso[0m
terminate called after throwing an instance of 'CORBA::COMM_FAILURE'
state_publisher: /usr/include/boost/thread/pthread/pthread_mutex_scoped_lock.hpp:26: boost::pthread::pthread_mutex_scoped_lock::pthread_mutex_scoped_lock(pthread_mutex_t*): Assertion `!pthread_mutex_lock(m)' failed.
testtest_hironx_ros_bridge_pose ... FAILURE!
FAILURE: test [test_hironx_ros_bridge_pose] did not generate test results
  File "/usr/lib/python2.7/unittest/case.py", line 327, in run
    testMethod()
  File "/opt/ros/hydro/lib/python2.7/dist-packages/rostest/runner.py", line 162, in fn
    self.assert_(os.path.isfile(test_file), "test [%s] did not generate test results"%test_name)
  File "/usr/lib/python2.7/unittest/case.py", line 420, in assertTrue
    raise self.failureException(msg)
--------------------------------------------------------------------------------
```

k-okada commented:
```
[addJointGroup] group name = larm
[addJointGroup] group name LARM is already installed
position limit over LARM_WRIST_R(25), qRef=7.63981e+162, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=7.7419e+167, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=1.53681e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=4.75332e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=7.99098e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.49175e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=9.51612e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=8.31801e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
position limit over LARM_WRIST_R(25), qRef=5.56927e+169, llimit =-1.309, ulimit =0.122173, servo_state = ON, prev_angle = 3.16202e-322
```
は「時々」初期化に失敗している系だろうか．．．

snozawa commented:
> は「時々」初期化に失敗している系だろうか．．．

そんなきがしますね。。。
時々である点と値がとんでいる点なので。
今色々とテストを足しているので、seqも追加してみたいと思います。
